ydb/docs/en/core/deploy/configuration/config.md

```diff
@@ -456,6 +456,60 @@ actor_system_config:
 | `progress_threshold` | The actor system supports requesting message sending scheduled for a later point in time. The system might fail to send all scheduled messages at some point. In this case, it starts sending them in "virtual time" by handling message sending in each loop over a period that doesn't exceed the `progress_threshold` value in microseconds and shifting the virtual time by the `progress_threshold` value until it reaches real time. |
 | `resolution` | When making a schedule for sending messages, discrete time slots are used. The slot duration is set by the `resolution` parameter in microseconds. |
 
+## Memory {#memory}
```

Comment:
For your information, this documentation section is undergoing a major refactoring. This patch mostly changes the old content that has yet to be adapted to the new structure. However, as far as I remember, no other open PRs are changing these particular files right now, so it seems fine to apply this PR and continue the refactoring later.